safe_transaction_service/history/models.py

```diff
@@ -1471,7 +1460,7 @@ def not_indexed_metadata_contract_addresses(self):
 
 class MultisigTransactionQuerySet(models.QuerySet):
     def ether_transfers(self):
-        return self.exclude(value=0)
+        return self.filter(value__gt=0)
```

Comment:
I'm not getting this, remove the index for value__gt=0 and start using this filter?

Reply:
This is for MultisigTransaction, not InternalTx